04_Running_The_Class.Rmd

```diff
@@ -0,0 +1,285 @@
+
+To create a Group:
+
+::::{.borrowed_chunk}
+```{r, echo = FALSE, results='asis'}
+cow::borrow_chapter(
+  doc_path = "child/_child_terra_group_create.Rmd",
+  repo_name = "jhudsl/AnVIL_Template"
+)
+```
+::::
+
+You now have a unique **Student Group**.
+
+### Add Instructors to Group
+
+Now that your student Group has been created, you should add any additional instructors / course coordinators as **Admins**.  This will give them permission to add and remove students, in case you are unavailable.
+
+::: {.notice}
+If you created an instructor Group, you can instead add the *Instructor Group* as an Admin of the *Student Group*.  This will enable everyone in the Instructor Group to act as an Admin for the Student Group.  To do this, you will need the email address associated with the Instructor Group, which can be found on the Group management page: https://anvil.terra.bio/#groups.
+:::
+
+The following instructions explain how to add someone to a Terra Group.  You will want to add instructors to the Student Group as **Admins**.
+
+:::: {.borrowed_chunk}
+```{r, echo = FALSE, results='asis'}
+cow::borrow_chapter(
+  doc_path = "child/_child_terra_group_add_member.Rmd",
+  repo_name = "jhudsl/AnVIL_Template"
+)
+```
+::::
+
+## Have Students Create Accounts
+
+Each of your students will need to:
+
+1. Create a Google account, if they do not already have one
+2. Sign in to Terra
+3. Provide you with their username, so that you can add them to your Student Group
+
+Student-facing instructions for account creation (and several other activities) are provided in the [**Student Instructions**](#student-instructions) section.
+
+## Add Students to Group
+
+Once your students have accounts, you will need to add them to your Student Group as **Members**.
+
+::: {.notice}
+In order to add students to your Group, you will need the usernames for the Google accounts your students are using to access AnVIL.
+:::
+
+Unfortunately there is currently no way to add members to a Group in bulk, so you will need to add students to the Group one-by-one.
+
+The following instructions explain how to add someone to a Terra Group.  You will want to add students to the Group as **Members**.
+
+:::: {.borrowed_chunk}
+```{r, echo = FALSE, results='asis'}
+cow::borrow_chapter(
+  doc_path = "child/_child_terra_group_add_member.Rmd",
+  repo_name = "jhudsl/AnVIL_Template"
+)
+```
+::::
+
+## Add Student Group to Workspace(s)
+
+Now that you have added students to the Group, you can easily share Terra resources with the entire Group.
+
+First, you will need to give your Student Group access to any Workspaces that they will need.
+
+If you are **using a Public Workspace** (most pre-made content uses Public Workspaces), **you can skip this step** - Public Workspaces are automatically available to everyone on AnVIL.  
```

Comment:
Isn't it still nice to share the public workspace so it shows up under "My Workspaces" on students' interface? Super minor point, and not sure if this falls under "too much info"

Reply:
Oh, can you share a public Workspace?  In that case maybe I'll just delete this paragraph.

Reply:
I don't seem to be able to share arbitrary public Workspaces.  So if somebody is using one of our Workspaces but wasn't involved in developing it then I don't think it's possible for them to share it, unless they want to clone their own copy...
On that note, cloning isn't hard, if we want to recommend doing that, and it would ensure that it stayed stable for them while they were teaching, so maybe we want to recommend that anyway, in which case we could drop this paragraph.

Opened #99 to follow up

Reply:
Yeah, you can only share a public workspace if you own (I think).. maybe let's not worry about sharing these :)